internal/bed/bed.go

```diff
@@ -12,16 +12,17 @@ var humanChrOrder = []string{"1", "chr1", "2", "chr2", "3", "chr3", "4", "chr4",
 // Note that the the user will give the columns with 1-based indexing,
 // but that we convert this to zero-based indexing in .VerifyAndHandle()
 type Bedfile struct {
-	Input  string `arg:"" help:"Bed file path"`
-	Output string `env:"OUTPUT_FILE" short:"o" help:"Path to the output file. If unset the output will be written to stdout"`
+	Inputs []string `arg:"" help:"Bed file path(s)"`
```

Comment:
maybe indicate how to separate them, I assume it's with a comma? People might get different ideas on how that is done 😉

Reply:
also having different inputs shifts the meaning of `merge` - for me it is less clear what is actually merged, is it files or regions or both.

Reply:
Updated in commit 5376c71de60904b3a85d938306084d4255f87c00